app/pages/Accounts/TransactionListElement.tsx

```diff
@@ -214,6 +221,42 @@ function statusSymbol(status: TransactionStatus) {
     }
 }
 
+const memoTypes = [
+    TransactionKindString.Transfer,
```

Comment:
It is confusing that `Transfer`, `EncryptedAmountTransfer` and `TransferWithSchedule` are part of `memoTypes` as they are not memo types.

Reply:
I have refactored the variable away.